app/src/main/java/de/grobox/transportr/locations/LocationFragment.java

```diff
@@ -131,7 +130,7 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 		linesLayout = v.findViewById(R.id.linesLayout);
 		linesLayout.setVisibility(GONE);
 		linesLayout.setAdapter(adapter);
-		linesLayout.setLayoutManager(new LinearLayoutManager(getContext(), HORIZONTAL, false));
+		linesLayout.setLayoutManager(new LinearLayoutManager(getContext(), RecyclerView.HORIZONTAL, false));
```

Comment:
What rule isn't allowing us to do a static import here?

Reply:
I guess the problem was that we included `HORIZONTAL` from a deprecated namespace in this place.

Reply:
So we can static import it from the new one again?

Reply:
I guess so. Is it better practice to do so?

Reply:
Changed.

Reply:
It is a matter of taste. In times of modern IDEs that show you where things come from, I prefer static imports as they make the code smaller and easier to read.